common/string.test.ts

```diff
@@ -0,0 +1,67 @@
+import { sanitizeString, escapeHtml } from './string';
+
+describe('sanitizeString', () => {
+    it('should remove HTML tags', () => {
+        expect(sanitizeString('<script>alert("XSS")</script>')).toBe(
+            '&lt;script&gt;alert(XSS)&lt;/script&gt;'
+        );
+        expect(sanitizeString('<div>Content</div>')).toBe(
+            '&lt;div&gt;Content&lt;/div&gt;'
+        );
+    });
+
+    it('should remove special characters', () => {
```

Comment:
Kan gjerne skrive noen enkle tester for alle tegn som blir fjernet så man slipper å tolke regex-en